core/src/main/resources/jenkins/install/pluginSetupWizard.properties

```diff
@@ -56,3 +58,17 @@ installWizard_pluginInstallFailure_message=Some plugins failed to install proper
 with the failed plugins
 installWizard_continue=Continue
 installWizard_retry=Retry
+installWizard_upgradePanel_title=Upgrade
+installWizard_upgradePanel_banner=Welcome to Jenkins 2!
+installWizard_upgradePanel_message=Jenkins 2 includes some great new features we think you'll love to use. Get your instance updated to the latest \
+and greatest plugins available and take advantage of them!
+installWizard_upgradePanel_recommendedActionTitle=Install Recommended Plugins
+installWizard_upgradePanel_recommendedActionDetails=Install the recommended plugins
+installWizard_upgradePanel_skipRecommendedPlugins=No thanks
+installWizard_upgradeComplete_title=Upgrade
+installWizard_pluginsInstalled_banner=Welcome to Jenkins 2!
+installWizard_upgradeComplete_message=Congratulations! You've upgraded to Jenkins 2.</p><p>To learn more about the great new features, visit <a target="_blank" href="https://jenkins.io/2.0/">the Jenkins 2 website</a>!
+installWizard_upgradeSkipped_title=Upgrade
+installWizard_upgradeSkipped_banner=Features Not Installed
+installWizard_upgradeSkipped_message=<div class="alert alert-warning fade in">You've decided to skip updating to the great features in Jenkins 2.</div></p><p><a target="_blank" href="https://jenkins.io/2.0/">Click here to learn what you're missing</a>!
```

Comment:
> You've decided to skip updating to the great features in Jenkins 2. 


I suggest just dropping this sentence, it comes across almost patronizing to me. Just the click prompt should be sufficient IMO